sponsorblockcast.sh

```diff
@@ -28,6 +32,19 @@ watch () {
     if echo "$status" | grep -q "YouTube (PLAYING)"
     then
       video_id=$(echo "$status" | grep -oP "id=\"\K[^\"]+")
+      video_title=$(echo "$status" | grep -oP "title=\"\K[^\"]+")
+      video_artist=$(echo "$status" | grep -oP "artist=\"\K[^\"]+")
+      if [ -z "$video_id" ]
```

Comment:
Check if the API key is defined here too, else there's no point calling the API

Reply:
Good call. 531a48ac2c95aaa229d3272935cc5078f2090b5d